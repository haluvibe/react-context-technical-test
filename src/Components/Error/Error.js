import React from 'react';

const Error = () => <div>Oops, something went wrong whilst fetching the punk api...</div>;

export default Error;
