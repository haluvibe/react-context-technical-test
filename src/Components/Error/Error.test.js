import React from 'react';
import { shallow } from 'enzyme';
import Error from './Error';

describe('Error', () => {
  it('should render correctly with no props', () => {
    const wrapper = shallow(<Error />);

    expect(wrapper).toMatchSnapshot();
  });
});
