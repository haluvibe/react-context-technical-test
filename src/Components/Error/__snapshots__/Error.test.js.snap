// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should render correctly with no props 1`] = `
<div>
  Oops, something went wrong whilst fetching the punk api...
</div>
`;
